src/plugins/telemetryTable/TelemetryTableRow.js

```diff
@@ -88,6 +89,15 @@ define([], function () {
     getContextMenuActions() {
       return ['viewDatumAction', 'viewHistoricalData'];
     }
+
+    updateWithDatum(updatesToDatum) {
+      const normalizedUpdatesToDatum = createNormalizedDatum(updatesToDatum, this.columns);
+      this.datum = {
+        ...this.datum,
+        ...normalizedUpdatesToDatum
+      };
+      this.fullDatum = this.datum;
+    }
```

Comment:
I think if we emitted an event here when an in-place update happens, the Vue component could just update itself in-place and avoid a lot of extra work in `table.vue`

Reply:
Are you referring to older code here? There were no new changes to table.vue.

Reply:
My previous comment is resolved, it wasn't relevant.